test/spec/smartbanner_spec.js

```diff
@@ -110,6 +110,74 @@ describe('SmartBanner', function() {
 
     });
 
+    context('direct api usage', function() {
+      const HTML_WITH_API = `<!doctype html>
+        <html style="margin-top:10px;">
+        <head>
+          <meta charset="utf-8">
+          <meta name="smartbanner:api" content="true">
+        </head>
+        <body>
+        </body>
+      </html>`;
+
+      context('when we append meta api and no arguments in constructor', function() {
+          before(function() {
+            global.window = jsdom.jsdom(HTML_WITH_API, {userAgent: USER_AGENT_IPHONE_IOS9}).defaultView;
+            global.document = window.document;
+          });
+
+          it('expected to throw error', function() {
+            expect(() => new SmartBanner()).to.throw('No options detected in constructor with api enabled');
+          });
+      });
+
+      context('when on we do not append meta api with arguments in constructor', function() {
```

Comment:
Let's change to _when API disabled, but constructor parameters given_ to be clearer.